GVRf/Framework/framework/src/main/jni/objects/components/transform.cpp

```diff
@@ -34,6 +34,7 @@ Transform::Transform() :
 }
 
 Transform::~Transform() {
+    owner_object_->onTransformChanged();
```

Comment:
Looks that owner_object_ is not obligatory by Transform's constructor.
I think could be a good practice check if owner_object_ is not NULL before reference it just to avoid some future issue.

Reply:
Good catch!